components/locale-provider/index.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import * as moment from 'moment';
 import interopDefault from '../_util/interopDefault';
-import { ModalLocale, changeConfirmLocale } from '../modal/locale';
+import { ModalLocale, changeConfirmLocale } from '../Modal/locale';
```

Comment:
https://github.com/ant-design/ant-design/tree/master/components/modal

It is `modal`.

Reply:
https://unpkg.com/browse/antd@3.24.2/es/modal/